app/helpers/application_helper.rb

```diff
@@ -127,11 +127,16 @@ def link_to_function_if_authorized(name, function, options = {}, html_options =
   end
 
   def display_delete_if_authorized(options = {}, html_options = {})
+    return "" unless authorized_for(options)
```

Comment:
I need to display the link if either the user is authorized _or_ `@editing_self` is true.
I think, the change is quite harmless.
Before this commit, `display_delete_if_authorized` would just call `display_link_if_authorized`. The latter just calls `link_to` if `authorized_for` returns a truthy value or `""`.
Now this logic is also present in `display_delete_if_authorized`. Any better ideas are greatly appreciated, though?